src/components/LinkOut.tsx

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+
+interface LinkOutProps {
+    label: string | React.ReactNode;
+    href: string;
+    newTab?: boolean;
+}
+
+export const LinkOut = ({label, href, newTab}: LinkOutProps) => {
```

Comment:
nit: since this is called LinkOut, and you have an optional prop, I would think the default behavior is to have it go out. 

Also, right now the right hand links are not linking out, and I'm pretty certain they're supposed to be because they have the external icon (sorry if I put my comment about the left hand side not linking out in the wrong line and you changed it because of my comment). It can also be confusing to have an optional boolean because you don't know you're making a choice by not sending a prop in 
 
my suggestions: 
1. call this LinkOut, and have an optional `samePage` disable the default `target=_blank` behavior 
2. have this just called Link, and make a required prop that's an enum of either `SamePage` `NewTab` so you have to explicitly choose